packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_flutter_web.dart

```diff
@@ -35,7 +35,10 @@ class GoogleMapsPlugin extends GoogleMapsFlutterPlatform {
 
   @override
   Future<void> init(int mapId) async {
-    _map(mapId).init();
+    // The internal instance of our controller is initialized eagerly in `buildView`,
+    // so we don't have to do anything in this method, which is left intentionally
+    // blank.
+    assert(_map(mapId) != null, 'Must call buildWidget before init!');
```

Comment:
This assertion is added to preserve some of the validation that happened before, mainly that calling init before buildWidget would cause quite a lot of breakage.

This method used to call `init` in the controller, but no longer. We do that immediately after creating the controller in buildView.